orcid-integration-test/src/test/java/org/orcid/integration/blackbox/api/v2/rc1/PublicV2Test.java

```diff
@@ -52,7 +52,7 @@
 import org.orcid.jaxb.model.common_rc1.Day;
 import org.orcid.jaxb.model.common_rc1.Month;
 import org.orcid.jaxb.model.common_rc1.Year;
-import org.orcid.jaxb.model.error.OrcidError;
+import org.orcid.jaxb.model.error_rc2.OrcidError;
```

Comment:
Hey @nitw-shobhit,

Before merging this, please update the error you are using on RC1 tests, look for all tests below /v2/rc1/ and use the  org.orcid.jaxb.model.error_rc1.OrcidError instead of the org.orcid.jaxb.model.error_rc2.OrcidError

Thanks